sdk/servicebus/service-bus/src/core/messageSender.ts

```diff
@@ -78,7 +80,7 @@ export class MessageSender extends LinkEntity {
    * @private
    */
   private _sender?: AwaitableSender;
-  private _retryOptions?: RetryOptions;
+  private _retryOptions: RetryOptionsInternal;
```

Comment:
Followup - update this in the other places as well, like the receiver, sessionReceiver, and the managementClient.

Reply:
Agreed. :)